meta-flutter/README.md

```diff
@@ -1,22 +1,72 @@
 # meta-flutter for Yocto Project
-Recipe file examples for Yocto Project Yocto Project. See also: https://docs.yoctoproject.org/
+Recipe file examples for Yocto Project. See also: https://docs.yoctoproject.org/
```

Comment:
Recipe file examples for Yocto Project. In this README, we explain how to build for aarch64 using `core-image-weston` which is one of Yocto Images, and `dunfel` which is one of LTS Yocto versions. See also: https://docs.yoctoproject.org/

Reply:
I fixed it.